Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -11,28 +11,60 @@ class AuthError(Exception):
     pass
 
 
-def str_arg(args, name):
-    return str(args.get(name, ""))
+Null = object()
 
 
-def int_arg(args, name):
-    return arg_to_number(args.get(name, None), arg_name=name)
+def str_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return None
+    if nullable and arg == "null":
+        return Null
+    return str(arg)
 
 
-def bool_arg(args, name):
-    return argToBoolean(args.get(name, False))
+def int_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return None
+    if nullable and arg == "null":
+        return Null
+    return arg_to_number(arg, arg_name=name)
 
 
-def list_arg(args, name):
-    return argToList(args.get(name, []))
+def bool_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return None
+    if nullable and arg == "null":
+        return Null
+    return argToBoolean(arg)
+
+
+def list_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return None
+    if nullable and arg == "null":
+        return Null
+    return argToList(arg)
+
+
+def json_arg(args: Dict[str, Any], name: str, nullable=False):
+    arg = args.get(name, "")
+    if arg == "":
+        return None
+    if nullable and arg == "null":
+        return Null
+    return json.loads(arg)
 
 
 def add_key_to_outputs(outputs: dict, key_name: str, key_val):
     if type(outputs) is dict and key_name not in outputs:
         outputs[key_name] = str(key_val)
 
 
-def to_markdown(name, t):
+def to_markdown(name: str, t):
     try:
         return tableToMarkdown(name, t)
```

Comment:
Thank you for the information. I think that it's fine, it may help the user understand the full data structure he is working on. Most of the time, the Bastion does not return unset fields, so the presence of a null value is likely a kind of information.